ribbon-httpclient/src/main/java/com/netflix/niws/client/http/RestClient.java

```diff
@@ -282,15 +283,15 @@ protected Client apacheHttpClientSpecificInitialization() {
             }
         }
 
-        if (ncc.getProperty(CommonClientConfigKey.Linger) != null) {
+        if (ncc.get(CommonClientConfigKey.Linger) != null) {
             try {
                 HttpConnectionParams.setLinger(httpClientParams, ncc.getOrDefault(CommonClientConfigKey.Linger));
             } catch (Exception e) {
                 throwInvalidValue(CommonClientConfigKey.Linger, e);
             }
         }
 
-        if (ncc.getProperty(CommonClientConfigKey.ProxyHost) != null) {
+        if (ncc.get(CommonClientConfigKey.ProxyHost) != null) {
```

Comment:
Should all these ncc.get != null checks instead be ncc.containsKey or some such?

Reply:
I'm actually trying to get rid of containsKey so fine with the null check out here.